components/server/src/projects/projects-service.ts

```diff
@@ -115,8 +115,10 @@ export class ProjectsService {
     protected async onDidCreateProject(project: Project, installer: User) {
         let { userId, teamId, cloneUrl } = project;
         const parsedUrl = RepoURL.parseRepoUrl(project.cloneUrl);
-        if ("gitlab.com" === parsedUrl?.host) {
-            const repositoryService = this.hostContextProvider.get(parsedUrl?.host)?.services?.repositoryService;
+        const hostContext = parsedUrl?.host ? this.hostContextProvider.get(parsedUrl?.host) : undefined;
+        const type = hostContext && hostContext.authProvider.info.authProviderType;
+        if (type === "GitLab" || type === "Bitbucket") {
```

Comment:
@jldec, this is the change which treats all GItLab installations equal, so testing with gitlab.com should be enough.

Reply:
ok - i'll wait for staging to test :)